articles/sentinel/normalization.md

```diff
@@ -107,6 +107,43 @@ Each schema field has a type. Some have built-in, Azure Log Analytics types such
 
 <br>
 
+### Common fields
+
+The following fields are common to all ASIM schemas. They are documented here, but also for each schema, as there might be unique details relevant to each schema. For example the valid values for EventType or the valud of EventSchemaVersion. 
+
+| Field               | Class       | Type       |  Description        |
+|---------------------|-------------|------------|--------------------|
+| <a name="timegenerated"></a>**TimeGenerated** | Built-in | datetime | The time the event was generated by the reporting device.|
+| **_ResourceId**   | Built-in |  guid     | The Azure Resource ID of the reporting device or service, or the log forwarder resource ID for events forwarded using Syslog, CEF or WEF. |
+| **EventMessage**        | Optional    | String     |     A general message or description, either included in or generated from the record.   |
+| **EventCount**          | Mandatory   | Integer    |     The number of events described by the record. <br><br>This value is used when the source supports aggregation, and a single record may represent multiple events. <br><br>For other sources, set to `1`.   |
+| **EventStartTime**      | Mandatory   | Date/time  |      If the source supports aggregation and the record represents multiple events, this field specifies the time the that first event was generated. <br><br>Otherwise, this field aliases the [TimeGenerated](#timegenerated) field. |
+| **EventEndTime**        | Mandatory   | Alias      |      Alias to the [TimeGenerated](#timegenerated) field.    |
+|  <a name=eventtyoe></a>**EventType**           | Mandatory   | Enumerated |    Describes the operation reported by the record. Each schema documents the list of values valid for this field. |
+| **EventSubType** | Optional | Enumerated | Describes a subdivision of the operation reproted in [EventType](#eventtype). Each schema documents the list of values valid for this field. |
```

Comment:
Describes a subdivision of the operation reported in the [EventType](#eventtype) field. Each schema documents the list of values valid for this field.